### DISCUSSION TITLE
Gitcoin

### DISCUSSION THREAD

**Discussion by mrkay100** [zksync/credo#99] [Ideas]
I feel like the whole guild stuff is not enough to get bots out because it does not strongly represent a good kyc. I feel like other ideas should be raised as I have seen more and more people successfully sybil gitcoin

**Comment by RexKizzy** [comment]
I can say that most of the tasks are cool but the points allocation should be looked into too 
Some of restricted in some country and as such points can't be collected

**Comment by kamranbash** [comment]
Galxy Passport should solve the problem, it requires liveness testing, also Gitcoin does have specific stamps which deal with the issue so like CIVIC ID and another few which require liveness testing.

**Comment by axelcazorla** [comment]
im not gitoin now !!! fuck!!..

**Comment by axelcazorla** [comment]
espero que zkera tenga en cuanta su comunidad las txs volume edad esatn el la blockchain saludos amigos grande cadena!!!
